java/vector/src/main/java/org/apache/arrow/vector/BaseNullableFixedWidthVector.java

```diff
@@ -543,21 +533,28 @@ public void loadFieldBuffers(ArrowFieldNode fieldNode, List<ArrowBuf> ownBuffers
     */
    public List<ArrowBuf> getFieldBuffers() {
```

Comment:
What's the difference of this function and 

```
public ArrowBuf[] getBuffers(boolean clear)
```
?

Reply:
@siddharthteotia What's your thoughts on this?

Reply:
Ping @siddharthteotia, can you explain on the difference between the two methods and how do we need both as public API?